Tensile/Source/lib/source/ContractionSolution.cpp

```diff
@@ -610,15 +642,14 @@ namespace Tensile
 
         KernelInvocation rv;
 
-        bool debug = Debug::Instance().printKernelArguments();
-        rv.args    = KernelArguments(debug);
+        rv.args = KernelArguments(T_Debug);
 
         rv.args.reserve(512, 64);
 
         rv.kernelName = outputConversionKernelName(problem, inputs, hardware);
 
-        rv.workGroupSize.x = 8;
-        rv.workGroupSize.y = 8;
+        rv.workGroupSize.x = 256;
```

Comment:
Total workgroup size and number of work groups changes here - not sure if there are any side effects of this.

Reply:
We modify kernel as well.
Using 1-D work makes global read in 1-D that can improves performance by avoid bank conflict.